### ISSUE TITLE
Notification crashes Brave on Linux

### ISSUE THREAD

**Issue by btlechowski** [brave/browser-laptop#14848]
### Steps to Reproduce

1. Go to https://www.bennish.net/web-notifications.html
2. Click 'Show'
3. Allow the notification

**Actual result:**
Crash

**Expected result:**
No crash

**Reproduces how often:**

always

### Brave Version
```
Brave: 0.23.72 
V8: 6.8.275.24 
rev: c3b1cac002d807409d8438dc72688d9ae553cea5 
Muon: 8.0.2 
OS Release: 4.13.0-21-generic 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Linux 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 68.0.3440.75
```
Unable to reproduce on Windows 10 by @GeetaSarvadnya 
Unable to reproduce on Mac by @LaurenWags 

**Reproducible on current live release:**
No. Tested 0.23.39 on Ubuntu 17.10

cc @bsclifton @kjozwiak @srirambv

**Comment by LaurenWags** [comment]
Reproduced on UbuntuVM.

**Comment by testinnplayin** [comment]
Hi,
New to OpenSource. Thought I'd take a peek at the Brave project and saw this issue.

I tested this on Ubuntu 18.04 and it did not crash my browser. Here are the specs for my browser:
```
Brave: 0.22.810 
V8: 6.7.288.43 
rev: 8f30eeba0c2d6cccf1920a360b4774f820adbc58 
Muon: 7.0.6 
OS Release: 4.15.0-29-generic 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Linux 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 67.0.3396.71
```

![screenshot from 2018-07-26 20-34-22](https://user-images.githubusercontent.com/15613045/43281206-64cbfc08-9113-11e8-9b05-dae073e6000e.png)

I have a slightly older version of Brave but I hope this is still helpful. Please let me know if I can be of any help.

**Comment by riastradh-brave** [comment]
WFM


Name | Version
-- | --
Brave | 0.23.39
V8 | 6.7.288.46
rev | 3ee14b0
Muon | 7.1.6
OS Release | 4.13.0-46-generic
Update Channel | Release
OS Architecture | x64
OS Platform | Linux
Node.js | 7.9.0
Tor | 0.3.3.8 (git-99f9816efa08ab83)
Brave Sync | v1.4.2
libchromiumcontent | 67.0.3396.103

**Comment by bsclifton** [comment]
Hi there @testinnplayin 😄 Thanks for checking

This appears to be specific to Chromium 68, which we're in the process of updating to. This also explains why @riastradh-brave doesn't see it either.

For reference, here's my about:brave:
 
Brave: 0.23.72 
V8: 6.8.275.24 
rev: c3b1cac002d807409d8438dc72688d9ae553cea5 
Muon: 8.0.2 
OS Release: 17.7.0 
Update Channel: Release 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 68.0.3440.75

**Comment by darkdh** [comment]
this is C68 upgrade only problem

**Comment by darkdh** [comment]
```
Crash reason:  SIGILL                                                                                                                                                              
Crash address: 0x38990dd                                                                                                                                                           
Process uptime: not available                                                                                                                                                      
                                                                                                                                                                                   
Thread 0 (crashed)                                                                                                                                                                 
0  brave!brightray::LibnotifyNotification::Initialize() [libnotify_notification.cc : 71 + 0x0]
1  brave!brightray::NotificationPresenter::Create() [notification_presenter_linux.cc : 14 + 0x5]
2  brave!brightray::BrowserClient::GetNotificationPresenter() [browser_client.cc : 50 + 0x5]
3  brave!brave::(anonymous namespace)::OnWebNotificationAllowed(brightray::BrowserClient*, SkBitmap const&, content::PlatformNotificationData const&, brightray::NotificationDelegate*, bool) [platform_notification_service_impl.cc : 44 + 0x5]
4  brave!brave::BravePermissionManager::OnPermissionResponse(int, GURL const&, base::RepeatingCallback<void (std::__1::vector<blink::mojom::PermissionStatus, std::__1::allocator<blink::mojom::PermissionStatus> > const&)> const&, std::__1::vector<content::PermissionType, std::__1::allocator<content::PermissionType> > const&, std::__1::vector<blink::mojom::PermissionStatus, std::__1::allocator<blink::mojom::PermissionStatus> > const&) [callback.h : 125 + 0x5]  
```
it crashes after dlsym setup `dlsym(library_, "notify_is_initted"));` for `libnotify.so.4`
and access the method it load
`libnotify_loader_.notify_is_initted()`

**Comment by testinnplayin** [comment]
Ok got the beta version. It does indeed crash on Ubuntu 18.04 as well

```
Brave: 0.23.72 
V8: 6.8.275.24 
Muon: 8.0.2 
OS Release: 4.15.0-29-generic 
Update Channel:  
OS Architecture: x64 
OS Platform: Linux 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 68.0.3440.75
```

**Comment by darkdh** [comment]
fixed by https://github.com/brave/browser-laptop-bootstrap/commit/eeeeff9af860508bf8319b29ca00d5fafc595756
will be available in muon 8.0.3

**Comment by btlechowski** [comment]
Thanks @darkdh. Great work!

Verified on Ubuntu 17.10 x64
* 0.23.73 50bdb6df42550dd14f5636770ec8585aa26e361b
* Muon 8.0.3
* libchromiumcontent 68.0.3440.75
